src/containerapp/azext_containerapp/custom.py

```diff
@@ -175,17 +211,288 @@ def update_containerapp_yaml(cmd, name, resource_group_name, file_name, from_rev
         handle_raw_exception(e)
 
 
-def create_containerapp_yaml(cmd, name, resource_group_name, file_name, no_wait=False):
-    yaml_containerapp = process_loaded_yaml(load_yaml_file(file_name))
+def _override_parameters_into_containerapp(cmd,
```

Comment:
Can you add a comment to explain purpose of the helper method. With the definition of the method here , whenever a new property is added to the ContaineApp object we need to update this? Is there a way to make this generic or is this a limitation because we don't use SDK to get the Model object defintion?

Reply:
Removed yaml from this release